config/config.go

```diff
@@ -10,10 +10,12 @@ import (
 // Configuration is stuff that can be configured externally per env variables or config file (config.yml).
 type Configuration struct {
 	Server struct {
-		Port int `default:"80"`
-		SSL  struct {
+		ListenAddr string `default:""`
+		Port       int    `default:"80"`
+		SSL        struct {
```

Comment:
I'm thinking about removing this functionality, most people add tls via a reverse proxy. What do you think?

Reply:
Maybe we can consider this later in another issue.

As for me, I am currently using this feature, as it is much easier to configure for modern TLS1.3 connections, but I am also OK with configuring SSL through reverse proxy.